docs/core-concepts/networking.md

```diff
@@ -0,0 +1,177 @@
+# Network Model
+
+Kontena's network model is based on the **Grid**, spanning a set of host **Nodes**.
```

Comment:
Attempted to edit these section links in, but blind as to if I got the syntax right for the website.

The section id's are different on GitHub vs the website..